### ISSUE TITLE
Visualizer and draw_sphere  are missing

### ISSUE THREAD

**Issue by Tianhuanyu** [cmower/optas#102]
Based on default branch, the example.py can not be run. 
The error code is shown here:

```
Traceback (most recent call last):
  File "optas/example/example.py", line 66, in <module>
    vis = optas.RobotVisualizer(robot, q=q_solution, params=params)  # solution
AttributeError: module 'optas' has no attribute 'RobotVisualizer'
```
However, I don't find these classes in optas.

**Comment by cmower** [comment]
Thanks for pointing this out @Tianhuanyu. I have recently re-vamped OpTaS's internal visualizer and clearly I forgot to update this example script.

**Comment by cmower** [comment]
Fixed in #103
